smartsnippets/models.py

```diff
@@ -9,6 +9,7 @@
 
 from cms.models import CMSPlugin
 
+import re
```

Comment:
Shouldn't mix stdlib imports with django or other third party lib imports.
See pep8 for how imports should be ordered: https://www.python.org/dev/peps/pep-0008/#imports